.globalconfig

```diff
@@ -0,0 +1,18 @@
+is_global = true
+
+# Globalization rules
+dotnet_code_quality.CA1305.excluded_symbol_names = T:System.Byte|T:System.SByte|T:System.Int16|T:System.UInt16|T:System.Int32|T:System.UInt32|T:System.Int64|T:System.UInt64|T:System.String|T:System.Text.StringBuilder|T:System.Convert
+
+dotnet_diagnostic.CA1305.severity = error
+dotnet_diagnostic.CA2101.severity = suggestion
+
+# Performance rules
+dotnet_code_quality.CA1826.exclude_ordefault_methods = true
+
+dotnet_diagnostic.CA1805.severity = silent
+dotnet_diagnostic.CA1822.severity = silent
```

Comment:
Personally I would use this one at Error level, marking members as static is a safe performance win in C# and it's also good at finding places where there's a bug because the method should actually be referencing something on the class but because it isn't the suggestion is there to make it static.

Reply:
I agree it could be higher, but the amount of current violations and fixes required made me ignore it for now.

Also note that `silent` implies that the rule *should* not be ignored but is currently.

Reply:
AH that silent is good to know thanks